clevertap-core/src/main/java/com/clevertap/android/sdk/network/api/CtApi.kt

```diff
@@ -184,27 +167,20 @@ internal class CtApi(
             return false
         }
 
-        if (isViewedEvent) {
-            if (spikyProxyDomain.isNotNullAndBlank()) {
-                return false
-            }
-        } else {
-            if (proxyDomain.isNotNullAndBlank()) {
-                return false
-            }
+        val toCheckProxy = if (isViewedEvent) { spikyProxyDomain } else { proxyDomain }
+        if (toCheckProxy.isNotNullAndBlank()) {
+            clearCachedHandshakeDomain.invoke()
```

Comment:
This lambda is non-null, so you can directly call it: ```clearCachedHandshakeDomain()```.

Reply:
removed this bit in chained PR as it is no longer needed.